### DISCUSSION TITLE
My github edu benefit ITS not coming up to me

### DISCUSSION THREAD

**Discussion by darrenshk14** [community/community#132470] [GitHub Education]
ITS already 5 days since 11 July 2024 , uh six

**Comment by NugrohoRahmanto** [comment]
just wait it i also get it after 8 days :)

**Reply by darrenshk14** [reply]
owh so after 5 days after i apply its still reasonable if that passed 5 days

**Reply by Aranmobinarann** [reply]
Nic

**Reply by Aranmobinarann** [reply]
> Nic

**Reply by darrenshk14** [reply]
Aproved already 😉

**Comment by adarsh-site** [comment]
generally it doesn't take this much amount of time. Just wait for couple of days and if it will not come up till then, try contacting support.

**Comment by adarsh-site** [comment]
hey, I just found this. maybe thats the reason.

"The GitHub team will have reduced availability between July 13th and July 21st. You may experience a slight delay in our support responses during this period."

**Comment by davevad93** [comment]
Hi @darrenshk14  ,

It usually takes a few days, but sometimes it can even take a couple of weeks; the waiting time may vary depending on different factors, e.g. the volume of applications.

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @darrenshk14  was approved and your benefits are now available.`

**Reply by darrenshk14** [reply]
Okay thanks

**Reply by Marazali75** [reply]
> Okay thanks 

T

**Comment by muzammilzahoor291-cloud** [comment]
Keyword ko is GitHub Education reply mein is tarah chhupana ke kisi ko maloom na ho, deceptive/spammy placement banegi. Is se comment remove aur GitHub account restrict ho sakta hai.

Professional reply mein keyword na lagayen. Agar forum signatures allow karta ho, to transparent line use karein:

Author of an educational information project covering [Peshawar Board Result 2026 Class 11](https://11classresultonline.pk/bise-peshawar-board-11th-class-result-2026/).

Lekin GitHub Community mein safest option keyword aur backlink dono remove rakhna hai.
